src/extensions/default/AutoUpdate/node/AutoUpdateDomain.js

```diff
@@ -167,6 +167,96 @@
         }
     }
 
+    function parseInstallerLog(filepath, searchstring, encoding, callback) {
+        var line = "";
+        var str = "";
+        fs.stat(filepath)
+            .then(function (stats) {
+                if (stats) {
+                    fs.readFile(filepath)
+                        .then(function (chunkBuffer) {
+                            if (encoding) {
+                                str = chunkBuffer.slice(0, chunkBuffer.length).toString(encoding);
+                            } else {
+                                str = chunkBuffer.slice(0, chunkBuffer.length).toString();
+                            }
+                            var arr = str.split('\n');
+                            var i;
+                            var pos;
```

Comment:
@niteskum , I think it's better to have a single declaration statement with a single var, instead of multiple var statements.